src/sysfiles.c

```diff
@@ -3598,6 +3598,56 @@ Obj SyReadStringFid(Int fid) {
 #endif
 
 
+#ifdef SYS_IS_64_BIT
+void * SyMemmove(void * dst, const void * src, UInt size)
+{
+    return memmove(dst, src, size);
+}
+#else
+// The memmove in glibc on 32-bit SSE2 systems, contained in
+// __memmove_sse2_unaligned, is buggy in at least versions
+// 2.21 - 2.25 when crossing the 2GB boundary, so GAP must
+// include it's own simple memmove implementation.
```

Comment:
"it's" -> "its"

Perhaps also link to the glibc bug report. Oh, and they just release 2.26 which also has the bug.